README.md

```diff
@@ -14,6 +14,9 @@ running the responsive public website and set `development` to `true`
 to always return `true` in public_website or responsive_site
 4. Run `bundle exec jekyll serve --watch`
 
+The test harness uses a key pointing to one of the partner tools. 
+In order to test a partner tool in the test environment, set the key to point to the relevant partner tool here: https://github.com/moneyadviceservice/test-harness/blob/gh-pages/_config.yml#L4
```

Comment:
Hey @Tr4pSt3R thank you for the suggestion, but since I already merged the PR, can you open a PR with the proposed change? I will thumb because the description is better. :)